packages/snap-store-mobx/src/Search/Stores/SearchPaginationStore.ts

```diff
@@ -184,6 +211,29 @@ export class SearchPaginationStore {
 	}
 }
 
+export class PageSizeOption {
+	public services: StoreServices;
```

Comment:
I think the services ought to be private - but I see it is not for the Page class currently... :(